src/common/components/notification-modal/notificationModal-components/AlertCardModal.tsx

```diff
@@ -0,0 +1,64 @@
+import { useRef, useEffect } from "react";
+import classNames from "classnames/bind";
+import IC_CLOSE from "@/images/ic_close.svg";
+import GetAlertCardModalData from "@/shared/hooks/getAlertCardModalData";
+import CreateCards from "./CreateCards";
+import styles from "./AlertModal.module.scss";
+
+const cn = classNames.bind(styles);
+
+export interface AlertModalProps {
```

Comment:
컴포넌트 명을 그대로 따라가주면 좋을 것 같아요~
```suggestion
export interface AlertCardModalProps {
```